### PR TITLE
Elemental resist/hp adjustment.

### PR DESCRIPTION
**Pull request by skittles1**
Adjusted elemental resistances, fixed Constructed issue with overwriting resistances, raised champions from level 230 to 250 and difficulty 8 to 9. Raised normal elementals from difficulty 7 to 8.

To address the individual resistances:
Earth Champion and Earth Elemental have 50% resist to all spells and all weapons. Weak to acid by 70%. Weak to cold and fire by 25%. Weak to bludgeon by 50%.

Nerudite Champion has 50% resist to all spells and all weapons. Has 60% weakness to quake, and 60% weakness to pierce.
Nerudite Elemental has 50% resist to all spells and all weapons. Has 70% weakness to quake, 60% weakness to pierce, and 20% weakness to holy and unholy.

Fire Champion has 90% resist to all spells, 60% to all weapons. Has 80% weakness to cold, and 50% weakness to thrust.
Fire Elemental has 70% resist to all spells, 60% to all weapons. Has 60% weakness to cold, and 50% weakness to thrust.

Ice Champion has 90% resist to all spells, 60% to all weapons. Has 80% weakness to fire, and 50% weakness to slash.
Ice elemental has 70% resist to all spells, 60% to all weapons. Has 60% weakness to fire, and 50% weakness to slash.

To sum up: Earth elementals are 'easiest', can be fought using acid and blunt weapons easily, and with cold/fire if necessary. Nerudite are slightly harder but can still be fought with quake or pierce damage. Fire and ice elementals are hardest and have more innate spell/weapon resist, but can be fought with the opposite element or thrust and slash weapons respectively. As discussed in IRC the resists are lower but difficulty (and level on the champions) is higher.

Keen, please let me know if this is what you meant in the earlier discussion :)

### CODE REVIEW

**Comment by treymd** [comment]
I think this is overall a better solution, it keeps the elements important while addressing some of the other concerns.
